package.py

```diff
@@ -0,0 +1,32 @@
+name = "oiio"
+
+version = "3.5.82_spiArn"
+
+authors = [
+    "lg"
+]
+
+
+description = \
+"""
+The primary target audience for OIIO is VFX studios and developers of tools such 
+as renderers, compositors, viewers, and other image-related software you'd find 
+in a production pipeline.
+"""
+
+tools = [
```

Comment:
We should Rez-ify the libraries as well.

Reply:
The python2.7 version of OpenImageIO is currently being built into site-packages directory. 

The tools list is a manifest of executables to be exported into a rez suite. The names should match file located in $PATH. In the commands function below, the path is defined as the package root/bin directory. 

I need to understand better how c++ compiles against python 2.7 and 3.7. We could be able to set environmental variables in package.py based on which to python rez package has resolved in the build step.

Reply:
I have updated the package.py to have more build environment variables. I have been able to build a simple c++ qt5 application using this new configuration.